src/applications/vaos/components/tests/PhoneLayout.unit.spec.js

```diff
@@ -33,6 +33,78 @@ describe('VAOS Component: PhoneLayout', () => {
     },
   };
 
+  describe('When appointment information is missing', () => {
+    it('should not display heading and text for empty data', async () => {
+      // Arrange
+      const store = createTestStore(initialState);
+      const appointment = {
+        comment: 'This is a test:Additional information',
+        location: {
+          stationId: '983',
+          clinicName: 'Clinic 1',
+          clinicPhysicalLocation: 'CHEYENNE',
+        },
+        videoData: {},
+        practitioners: [
+          {
+            identifier: [
+              {
+                system: 'dfn-983',
+                value: '520647363',
+              },
+            ],
+            name: {
+              family: 'NADEAU',
+              given: ['MARCY'],
+            },
+            practiceName: 'Cheyenne VA Medical Center',
+          },
+        ],
+        vaos: {
+          isCommunityCare: false,
+          isCompAndPenAppointment: false,
+          isCOVIDVaccine: false,
+          isPendingAppointment: false,
+          isUpcomingAppointment: true,
+          isPhoneAppointment: true,
+          isCancellable: true,
+          apiData: {},
+        },
+        status: 'booked',
+      };
+
+      // Act
+      const screen = renderWithStoreAndRouter(
+        <PhoneLayout data={appointment} />,
+        {
+          store,
+        },
+      );
+
+      // Assert
+      expect(
+        screen.getByRole('heading', {
+          level: 1,
+          name: /Phone appointment/i,
+        }),
+      );
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /Who/i,
+        }),
+      ).not.to.exist;
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /What/i,
+        }),
+      ).not.to.exist;
```

Comment:
Also need to add assertions for the text for missing facility/clinic info as well as the reason/other details texts.

Reply:
@JunTaoLuo Added assertions for the phone appointments